catalog/controller/extension/payment/razorpay.php

```diff
@@ -212,7 +211,15 @@ protected function orderPaid(array $data)
         $merchant_order_id = $data['payload']['payment']['entity']['notes']['opencart_order_id'];
         $razorpay_payment_id = $data['payload']['payment']['entity']['id'];
         if(isset($merchant_order_id) === true)
-        {    
+        {   
+            $this->load->model('extension/payment/razorpay');
```

Comment:
check if this can be loaded one time only in class.

Reply:
@ramth05 have checked for this, tried using class instance but it not working out.